docs/workflow_solutions/getting_containers.md

```diff
@@ -287,6 +287,66 @@ python python_test.py
 
 More lessons on Docker can be found in this link: [Introduction to Docker](https://christinalk.github.io/docker-introduction/) and [Docker Documentation](https://docs.docker.com/reference/dockerfile/).
 
+## How to Access and Use Community Containers in the GitLab Container Registry
+
+In the UAB Research Computing [GitLab instance](../account_management/gitlab_account.md#uab-gitlab-registration), we provide certain community containers—prebuilt, these containers are standardized environments that grant you access to a variety of software tools. in the sections below, we share information for what you will find in the gitlab community container repo, and how you can access them.
```

Comment:
What about "provide prebuilt community containers" and then end the sentence there? Then continue with the remainder.

There is also a typo with lowercase start of sentence "in the sections...". No need for commas in that sentence, either. Technically, we're not referring to a single repo, but a project with multiple repos. You may want to reword the sentence to allow for that change.